src/Unleash/Internal/UnleashServices.cs

```diff
@@ -174,6 +174,10 @@ public void Dispose()
             }
 
             engine?.Dispose();
+            if (scheduledTaskManager != null && !(scheduledTaskManager is SystemTimerScheduledTaskManager))
+            {
+                Logger.Warn(() => $"UNLEASH: Disposing ScheduledTaskManager of type {scheduledTaskManager.GetType().Name}");
```

Comment:
Should this be a warn? This kind feels like info to me

Reply:
We're disposing something not created by us, that could have side effects on their side?